assets/js/components/KeyMetrics/ConversionReportingDashboardSubtleNotification.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * ConversionReportingDashboardSubtleNotification component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useSelect } from 'googlesitekit-data';
+import { Button, SpinnerButton } from 'googlesitekit-components';
+import StarFill from '../../../svg/icons/star-fill.svg';
+import SubtleNotification from '../../googlesitekit/notifications/components/layout/SubtleNotification';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import Link from '../Link';
+
+export default function ConversionReportingDashboardSubtleNotification( {
+	ctaLabel,
+	handleCTAClick,
+	isSaving,
```

Comment:
`isSaving` is not required, we need to set the default value for it:
```suggestion
	isSaving = false,
```